get_molecule_xyz/get_molecule_xyz.py

```diff
@@ -7,7 +9,7 @@
 # 
 ####################################################################################################
 
-def cas_to_xyz(xyz_title:str,cas_number:str) -> None:
+def cas_to_xyz(xyz_title:str,cas_number:str) -> str:
     """ creates an xyz file from a given cas_number if the xyz file is available
 
     Parameters
```

Comment:
add documentation for the specified format of the cas number - are hyphens allowed?